### ISSUE TITLE
missing BASE-MIMIC.csv and BASE-MIMIC.csv

### ISSUE THREAD

**Issue by anothersin** [YH0517/AFLoc#1]
Hi,

Very interesting work.

But the lack of processing code for ['MIMIC-224-inter-area'](https://github.com/YH0517/AFLoc/blob/3890ad296769a7b813d88fa5cecbb9c6781ae656/afloc/constants.py#L12") and ['BASE-MIMIC.csv'](https://github.com/YH0517/AFLoc/blob/3890ad296769a7b813d88fa5cecbb9c6781ae656/afloc/constants.py#L13") makes it hard to follow.

Can you please provide the corresponding code to get them?

**Comment by YH0517** [comment]
We have provided the code for generating these files. Please see the "Preprocess" section in the README file.

**Comment by anothersin** [comment]
> We have provided the code for generating these files. Please see the "Preprocess" section in the README file.

Thanks, we are running code for pretraining now!
